Description: Skip test_fetch_package_version
 This test doesn't work in the pybuild environment.
Author: Gilles Dubuc <gilles@wikimedia.org>

Index: raven-5.18.0/tests/versioning/tests.py
===================================================================
--- raven-5.18.0.orig/tests/versioning/tests.py
+++ raven-5.18.0/tests/versioning/tests.py
@@ -34,6 +34,7 @@ def test_fetch_git_sha():
     ).decode('latin1').strip()
 
 
+@pytest.mark.skipif('True')
 def test_fetch_package_version():
     result = fetch_package_version('raven')
     assert result is not None
